docs/installation/installing-tentacles/polling-tentacles-web-sockets.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Polling Tentacles over WebSockets
+description: Octopus Polling Tentacles open a connection to the Octopus server over WebSockets to ask what to do.
+position: 1
+version: "3.12"
+---
+
+Read about [(TCP) Polling Tentacles](polling-tentacles.md) before continuing.
```

Comment:
As per slack chat, probably worth mentioning you need a certificate setup right at the start - https://octopusdeploy.slack.com/archives/D1V30964E/p1490849133528761